index.php

```diff
@@ -59,11 +59,18 @@
 
 	$save_path = "{$FROTZ_SAVE_PATH}/{$session_id}.zsav";
 
-	#
+
+
+
+	handle_save_and_restore($command, $save_path, $session_id, $FROTZ_SLOT_SAVES_ENABLED, $FROTZ_SAVE_SLOTS, $FROTZ_SAVE_PATH);
+
+
+    #
 	# Check for restricted commands
 	#
 	switch ($command){
 		case 'save':
+
 		case 'restore':
```

Comment:
Wont we need to remove the `save` and `restore` checks here, as they'll conflict with your change above?